CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 24.9.1 [#770](https://github.com/openfisca/openfisca-core/pull/770)
+
+- Fix instant (period) for AmoutTaxScale.
+  - Unblock the factorization of the French's tax system https://github.com/openfisca/openfisca-france/pull/1213 (Pinel)
```

Comment:
This is somewhat contingent, I'm not sure it's worth mentioning in a Changelog.

In the line above I would rather take the user perspective and highlight that we are solving a rather serious problem, e.g. "Fix a bug that prevented amount scale parameters from being accessed in a formula".

Reply:
Thanks @Morendil for your suggestion, I've taken it into account.